code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -462,14 +465,54 @@
 	if(user.a_intent == INTENT_HARM)
 		return ..()
 	if(locked)
-		to_chat(user, "<span class='warning'>The maintenance panel is locked.</span>")
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
 		return TRUE // Must be true or we attempt to stab the bot
 
 	open = !open
 	I.play_tool_sound(src)
 	to_chat(user, "<span class='notice'>The maintenance panel is now [open ? "opened" : "closed"].</span>")
 	return TRUE
 
+/mob/living/simple_animal/bot/wrench_act(mob/living/user, obj/item/I)
+	if(user.a_intent != INTENT_HELP)
+		return..()
+	if(locked) //ID-Lock
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+	if(!open) //Screwdriver panel
+		to_chat(user, "<span class='warning'>The maintenance panel is screwed in place.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+	if((bot_type != SEC_BOT) && (bot_type != GRIEF_BOT)) //Should only affect security bots and griefsky
+		to_chat(user, "<span class='warning'>The internal components cannot be wrenched more securely.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+
+	I.play_tool_sound(src)
+	wrenched = !wrenched
+	to_chat(user, "<span class='notice'>The internal components are now [wrenched ? "wrenched" : "unwrenched"].</span>")
+	return TRUE
+
+// Override me please!
+/mob/living/simple_animal/bot/proc/disassemble()
+	qdel(src)
+
+/mob/living/simple_animal/bot/multitool_act(mob/living/user, obj/item/I)
+	if(user.a_intent != INTENT_HELP)
+		return..()
+	if(locked) //ID-Lock
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+	if(!open) //Screwdriver panel
+		to_chat(user, "<span class='warning'>The maintenance panel is screwed in place.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+	if((bot_type == SEC_BOT) || (bot_type == GRIEF_BOT)) //Should only affect security bots and griefsky
+		if(wrenched)
+			to_chat(user, "<span class='warning'>The internal components are wrenched in place.</span>")
+			return TRUE // Must be true or we attempt to bash the bot
```

Comment:
```suggestion
	if((bot_type == SEC_BOT) || (bot_type == GRIEF_BOT) && wrenched) //Should only affect security bots and griefsky
		to_chat(user, "<span class='warning'>The internal components are wrenched in place.</span>")
		return TRUE // Must be true or we attempt to bash the bot
```

Reply:
I am not sure why I did not think of simply adding it to the check as well. I'll commit and make sure that the `&& wrenched` does apply to both `bot_type` checks just in case. Thanks!